Android/AndroidManifest.xml

```diff
@@ -138,19 +139,30 @@
 
         <activity
             android:name=".activities.helpers.BluetoothDevicesActivity"
-            android:noHistory="true"
             android:excludeFromRecents="true"
             android:launchMode="singleTask"
+            android:noHistory="true"
             android:theme="@style/CustomDialogTheme"/>
 
         <activity
             android:name=".maps.providers.google_map.DownloadMapboxMapActivity"
-            android:noHistory="true"
             android:excludeFromRecents="true"
-            android:theme="@style/CustomActionBarTheme.Transparent"
-            android:launchMode="singleTask"/>
+            android:launchMode="singleTask"
+            android:noHistory="true"
+            android:theme="@style/CustomActionBarTheme.Transparent"/>
+
+        <activity
+            android:name=".activities.GeoTagActivity"
+            android:label="@string/geo_tag_label"
+            android:launchMode="singleTop"/>
+
+        <activity
+            android:name=".activities.GetRecapImageActivity"
+            android:label="Get Recap Image"
```

Comment:
@ne0fhyk Actually this entire activity is removed. I deleted it from the manifest